gallery/src/guidelines/ColorPalette/Swatcher.scss

```
@@ -47,3 +47,21 @@
 .gallery-swatch__hex {
   flex-basis: 7ch;
 }
+
+// Safari does not support gap in flex layouts so we need to use this query and margin for Safari.
+@media not all and (min-resolution:.001dpcm) {
+  @supports (-webkit-appearance:none) and (display:flow-root) {
+    // override a default .nx-card style
+    .nx-card__content > :not(:last-child) {
```

Doesn't need to be quite this complicated
```suggestion
    .nx-card__content > .gallery-swatch {
```